test/parallel/test-child-process-flush-stdio.js

```diff
@@ -8,10 +8,22 @@ const p = cp.spawn('echo');
 p.on('close', common.mustCall(function(code, signal) {
   assert.strictEqual(code, 0);
   assert.strictEqual(signal, null);
+  spawnWithReadable();
 }));
 
 p.stdout.read();
 
-setTimeout(function() {
-  p.kill();
-}, 100);
```

Comment:
I removed the timeout here because I'm quite wary of such things as I've seen tests with timeouts like this become flaky.